modules/ROOT/pages/api-search.adoc

```diff
@@ -0,0 +1,38 @@
+= Search Inside Your API Specification Dependencies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Because Anypoint Studio 7.4 enables you to import your API specifications from Exchange or Maven as dependencies of your project, those dependencies no longer locate as resources inside the `src/resources/api/` directory, but as ZIP file dependencies in your project's classpath. This change makes your API dependencies not searchable through the File Search option in Studio. To enable you to search for specific text inside your API dependencies, Studio introduces API Search.
+
+You can access the API Search menu from the taskbar at the top of the Anypoint Studio display, selecting *Search* > *Search...*, or by using the `control + H` keyboard shortcut.
+
+image::api-search.png[]
+
+. Type the text that you want to search inside your API dependencies. +
+* Use `\*` to search for any string. For example, `*-api-*` searches for strings containing `-api-`.
+* Use `?` to allow any character after a specific pattern. For example, `customer?` searches for both `customer` and `customers`.
+* Use `\` to escape any of the reserved literals used in search patterns. For example, if you want to search for strings containing `\*`, `?`, or `\` you must escape them: `\*`, `\?`, and `\\`.
+. If you want to specify a specific name patter for the files you want to search through, type the pattern in the *File name patterns* field.
++
+* Use `\*` to search for any string. For example, `*-api-*` searches inside any API specification containing `-api-` in its name.
+* Use `?` to allow any character after a specific pattern. For example, `customer?.raml` searches inside both `customer.raml` and `customers.raml`.
+* Use `!` to exclude certain filenames or patterns. For example, `!examples.raml` searches inside all files except `examples.raml`.
+. Click the *Choose...* button to filter per file type.
+. Select the type of API specification in your project.
++
+* *API dependencies* searches inside API specifications imported from Exchange or Maven.
+* *Local API Files* searches inside API specifications added from a local file, or Design Center.
```

Comment:
That comma after file seems weird. We should remove it or added it in the API Dependencies explanation to be consistent.